docs/blog/2020-01-15-website-uptime/index.md

```diff
@@ -0,0 +1,81 @@
+
+When your app is experiencing scaling issues, on the other hand, it usually instigates a whodunnit across the entirety of the infrastructure. Databases, caches, load balancers, proxies, routing and edge configurations, cloud infrastructure like AWS or Azure your team utilizes, third party scripts, plugins, and services -- everything comes under suspicion and is therefore guilty until proven innocent.
+
+Meanwhile your entire engineering organization grinds to a halt as everyone who could potentially find the key bottleneck starts looking for it, and everyone else starts rubbernecking because the unfolding drama is _way more interesting_ than whatever else they were working on.
+
+If you're building a website, puts an end to traffic-spike-induced downtime.
+
+Gatsby sites are simply files that can be pushed to a CDN like Cloudflare or Netlify or AWS Cloudfront. And CDNs are much less likely to go down.
+
+## A site that's always ready for your guests
+
+Brian Webster, founder/CEO of the LA-based, design-focused agency Delicious Simplicity, compares how Gatsby builds the site ahead-of-time to being a good party host.
+
+"We think of our websites sort of as a big event or a big party and everyone coming to visit it are our guests," says Webster. "So not pre-building and assembling web pages on the fly at request time would kind of be like the equivalent of setting up your party when your guests show up. Why would we doing that on their time?"
+
+"As site owners, it's becoming more and more our responsibility to say, 'Hey, how much of this stuff can we get done before our guests show up?' Especially when we know what they're going to come and look at. Which means if we're expecting 10 people at our party or 10 thousand people show up, we're equipped to scale."
+
+In a world where website visitors are a source of revenue or donations, Webster points out, time is money, and traffic spikes from publicity, email newsletter blasts, should be taken advantage of rather than worried about.
+
+Wester again:
+
+"We never want our best day to be our worst day -- the site going down, or becoming slow," he says. "But with [Gatsby], everything's prebuilt. Scale is cheap, easy. I think that's just really great for your best day to be your best day and to get as many of those donations as you can."
+
+## No more clearing caches and hoping for the best
+
+Brad Redding, co-founder from digital agency and analytics provider Elevar, has been building e-commerce sites for over a decade.
+
+"We've had clients that are used to that aren't technical, who aren't skilled in HTML or CSS," Redding says.
```

Comment:
I know this is a quote, but is there a word missing? And in other blogs we've tried to stay away from the term "non-technical", could we paraphrase instead?

Reply:
good catch. Changed it